BcSaaS/Import-BcConfigurationPackage.ps1

```diff
@@ -0,0 +1,161 @@
+﻿<# 
+ .Synopsis
+  Function for importing and applying configuration package to online tenant
+ .Parameter environment
+  Name of the environment inside the tenant in which you want to publish the Per Tenant Extension Apps
+ .Parameter companyName
+  Company Name to import the package to
+ .Parameter packageFile
+  File with a package to import. Package code inside file must be equal to packageCode parameter
+ .Parameter packageCode
+  Package will be created with specified code
+ .Parameter packageName
+  Package will be created with specified name
+ .Parameter packageLang
+  Package will be created with specified language id, en-US (1033) by default.
+ .Parameter importPackage
+  Enable the switch to import the package after upload
+ .Parameter applyPackage
+  Enable the switch to apply the package after import
+ .Parameter useNewLine
+  Add this switch to add a newline to progress indicating periods during wait.
+  Azure DevOps doesn't update logs until a newline is added.
+
+#>
+function Import-BcConfigurationPackage {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $bcAuthContext,
+        [Parameter(Mandatory=$true)]
+        [string] $environment,
+        [Parameter(Mandatory=$false)]
+        [string] $companyName,
+        [Parameter(Mandatory=$true)]
+        $packageFile,
+        [Parameter(Mandatory=$true)]
+        $packageCode,
+        [Parameter(Mandatory=$false)]
+        $packageName = "",

```

Comment:
packageName is auto-calculated in the other function using something like this:

```
            if (!$packageIdSpecified) {
                $packageInfo = Get-PackageInfoFromRapidStartFile $configPackage
                if ($packageInfo) {
                    $packageId = $packageInfo.Code
                }
                else {
                    $packageId = [System.IO.Path]::GetFileNameWithoutExtension($configPackage)
                    $packageId = $packageId.SubString(0, [System.Math]::Min(20, $packageId.Length))
                }
             }
```
and then the packageId is used as both name and code.
Is this wrong?